src/ui.ts

```diff
@@ -3,21 +3,51 @@ import './ui.css'
 
 let contentWelcome = document.getElementById('welcomeContent');
 let contentMain = document.getElementById('mainContent');
+let contentError = document.getElementById('errorContent');
 let contentCode = document.getElementById('codeField');
 let copyButton = document.getElementById('copyBtn');
+let errortext = document.getElementById('errorMessage');
+let backButton = document.getElementById('backBtn');
 
 document.getElementById('generate').onclick = () => {
   parent.postMessage({ pluginMessage: { type: 'generate-variables' } }, '*');
-
-  contentWelcome.classList.add('hide');
-  contentMain.classList.remove('hide');
 }
 
 onmessage = (event) => {
   if (event.type === 'message') {
     const { mainColor, colorList } = event.data.pluginMessage;
-    const code = new CodeGenerator().codeFromVariables(mainColor, colorList);
-    (<HTMLTextAreaElement>contentCode).value = code;
+    if (mainColor != null && colorList != null) {
```

Comment:
There's a cooler way to check `null` hehe (variable in `if` clause will become a truth value)
```suggestion
    if (mainColor && colorList) {
```